docs/sections/versions/baker-3-release-notes.md

```diff
@@ -0,0 +1,215 @@
+In these cases there is no need to inquire on Baker itself anymore.
+
+```scala tab="Scala"
+case class SensoryEventResult(
+                               sensoryEventStatus: SensoryEventStatus,
+                               eventNames: Seq[String],
+                               ingredients: Map[String, Value]
+) extends common.SensoryEventResult with ScalaApi
+```
+
+```java tab="Java"
+case class SensoryEventResult(
+                               sensoryEventStatus: SensoryEventStatus,
+                               eventNames: java.util.List[String],
+                               ingredients: java.util.Map[String, Value]
+) extends common.SensoryEventResult with JavaApi {
+
+  def getSensoryEventStatus: SensoryEventStatus = sensoryEventStatus
+
+  def getEventName: java.util.List[String] = eventNames
+
+  def getIngredients: java.util.Map[String, Value] = ingredients
+}
+```
+
+### New EventListener functions
+For the EventListener we have moved away from the annotation based approach.
+In this version you can add a EventListener function instead.
+See the [event listener section](http://localhost:8000/sections/reference/event-listener/) for more details.
+
+## Improved and simplified Runtime
+### Better performance
+Due to simplifying and refactoring the runtime we see a big increase in the load a Baker instance can handle.
+For our created [example project](https://github.com/ing-bank/baker/tree/master/examples/src)
+we saw the project could handle over 5 times the load when switching between version 2 and 3 of Baker.
+
+### New journal for stateless processes
+For stateless processes we used to advise to use the in-memory-journal plugin.
+This journal kept all processes in-memory until the actors where cleaned.
+For a true stateless process this is unnecessary so we created a new journal.
+This one just dumps the event sourcing messages.
+This should lower the memory footprint of stateless users.
+
+How to configure:
+```
+CONFIG
+# Custom journal plugin
+service.sink-journal {
+  # Class name of the plugin.
+  class = "com.ing.baker.runtime.akka.journal.SinkJournalWriter"
+  # Dispatcher for the plugin actor.
+  plugin-dispatcher = "akka.actor.default-dispatcher"
+}
+
+service.sink-snapshot-store {
+  # Class name of the plugin.
+  class = "com.ing.baker.runtime.akka.journal.SinkSnapshotStore"
+  # Dispatcher for the plugin actor.
+  plugin-dispatcher = "akka.persistence.dispatchers.default-plugin-dispatcher"
+}
+akka.persistence {
+  journal.plugin = "service.sink-journal"
+  snapshot-store.plugin = "service.sink-snapshot-store"
+}
+```
+
+### Support for async interactions
+It is now possible to create interactions that return their events in a Future or CompletableFuture.
+When using this Baker can close threads instead of blocking a thread.
+Which in turn again improves performance.
+The downside of using async interaction that some validations cannot be done anymore due to type erasure.
```

Comment:
"async interaction that some validations" => "async interaction is that some validations"